---
layout: page
title: research 
permalink: /research/
---

#### **Ongoing research**

**Strawberry poision frog genomics**

<img src="/figures/Bocas.JPG" alt="Solarte Oophaga pumilio" style="float:right;padding-left:25px;padding-right:25px;width:350px">


*Oophaga pumilio* is the scientific name of the strawberry pison frog. *Oophaga* comes from the greek *phagos* (eat) and *oon* (egg), this type of frog feeds their tadpoles with unfertilized eggs. *Pumilio* means pigmy in greek, which is very descriptive, these frogs are the size of my fingernails. *O. pumilio* is red throughtout its continental distribution. This is an aposematic signal to warn its potential predators of its toxicity. However there is an archipelago in Panama called Bocas del Toro where there is a lot of phenotypic variation. There are several island and almost each one of them has their own frog color (yellow, green, orange, red, blue, etc). I study their evolution and population history. How are different evolutionary forces acting on this polymorphism? What are the genes behind this trait? <br><br>

**Evaluating the genomic rescue in Florida Panthers**

<img src="/figures/puma.png" alt="Solarte Oophaga pumilio" style="float:left;padding-left:25px;padding-right:25px;width:330px">

Populations of puma *Puma concolor* a native feline from the Americas are suffering from genomic inbreeding and small population sizes. Humans have reduced their habitats and isolated the populations. Particularly, a population in Big Cypress National Preserve in Florida was very affected. In 1995, eight pumas from Texas were introduced to the population with the purpose of genomic rescue. I am interesed in understanding how successful the rescue was. Is the local genetic diversity from Big Cypress preserved? Is selection helping preserve variation from a specific ancestry in different parts of the genome?


**Lizard adaptation genomics**

<img src="/figures/lizarddrawing.jpg" alt="Solarte Oophaga pumilio" style="float:right;padding-left:25px;padding-right:25px;width:250px">

I do populations genetics of the multicolor toad-headed lizard (*Phrynochephalus versicolor*). This lizard is originary from the Gobi Desert in China and Mongolia. There are different populations that vary in how much pigment they have in their skin. Most of the populations have light brown/yellow skin, same color as the sand they live in. There is a population that is black, matching the black sand of the region where it lives. I study the genetic basis of this adaptation. 
- [ Population genomics of variegated toad-headed lizard Phrynocephalus versicolor and its adaptation to the colorful sand of the Gobi Desert](/files/publications/2022Phrynocephalus.pdf)
<br><br>

**Genomic adaptation to free diving in the Haenyeo**

When an activity is strenuous and essential for a populationâ€™s survival, selection might shape genomic adaptation to this type of livelihood. The Haenyeo are female divers from Jeju Island in South Korea that have been diving to harvest sea products for centuries without scuba gear. We hypothesized that selection might have allowed the Haenyeo to adapt to a lifestyle of free diving. In collaboration with Dr. Joo-Young Lee from Seoul National University, we took physiological measurements from the Haenyeo, other Jeju citizens, and individuals from Seoul. I generated genomic data for each participant with low-coverage sequencing and imputation using the Korea1K Project. With this dataset, I can infer ancestry, population history, and potential loci under selection that might underlie an adaptation to diving. We hope to compare these results with other free-diving populations in Asia.


#### **Previous research**

**Microfluidics and transcriptional memory**

<img src="/figures/microfluid.jpg" alt="Solarte Oophaga pumilio" style="float:left;padding-left:25px;padding-right:25px;width:220px">

During my undergrad I had the opportunity to work with [Robert Schneider](https://www.helmholtz-munich.de/ife/research/robert-schneider-chromatin-dynamics-and-epigenetics/research/index.html). I was working under the supervision of Poonam Bheda. We worked together on her transcriptional memory postdoc project. We induced the *GAL1* gene in yeast by adding galactose to the media where the yeast where growing. Then we repressed the expression by changing the media to glucose. A second time the galactose was added and the expression of the *GAL1* gene was faster and higher. Using a microfuidics set up we would run experiments where we could track single cells for hours as we induced and repressed them several times. We studied the inheritance of this transcriptional memory and some mutants that would result in gain or loss os memory. Articles we published associated to this research:

- [Microfluidics for single-cell lineage tracking over time to characterize transmission of phenotypes in *Saccharomyces cerevisiae*](/files/publications/2020StarProtocol.pdf)

- [Single-Cell Tracing Dissects Regulation of Maintenance and Inheritance of Transcriptional Reinduction Memory](/files/publications/2020Microfluidics.pdf)


**Evolution of sex chromosomes**

My first bioinformatics and fieldwork experiences were with [Diego Cortez](https://www.ccg.unam.mx/en/diego-cortez-quezada/). In fact, I was his first student ever. One of the projects that I worked on was testing male mutation bias in monotremes (mammals that lay eggs like the platypus). Comparing mutation rates at autosomal versus sex chromosomes we estimated whether there was male mutation bias. The X, Y, and autosomes spend different proportions of time in each germline, and since the male germline (sperm) replicates a lot more, there is more room for generating mutations. We tested whether the bias existed in this particular system (venom-producing, egg-laying mammal, 5 pairs of sex chromosomes).

- [Male mutation bias is the main force shaping chromosomal substitution rates in monotreme mammals](/files/publications/2017MaleMutationBiasDCortez.pdf) 

[jekyll-organization]: https://github.com/jekyll
